src/vaults/AsyncRequests.sol

```diff
@@ -150,8 +153,8 @@ contract AsyncRequests is BaseInvestmentManager, IAsyncRequests {
         );
 
         require(
-            _canTransfer(vaultAddr, owner, address(escrow), shares)
-                && _canTransfer(vaultAddr, controller, address(escrow), shares),
+            _canTransfer(vaultAddr, owner, ESCROW_HOOK_ID, shares)
+                && _canTransfer(vaultAddr, controller, ESCROW_HOOK_ID, shares),
```

Comment:
Just a side note: 

* IF we anuyways need to whitelist the escrow in order for this too work, why have the `ESCROW_HOOK_ID` at all?

Reply:
Seems feasible to me. @hieronx WDYT?

Reply:
To be able to identify redemption requests (https://github.com/centrifuge/protocol-v3/blob/main/src/vaults/interfaces/token/IHook.sol#L21)